src/aiortc/contrib/media.py

```diff
@@ -27,39 +28,34 @@
 config_path = os.environ.get('CONFIG_PATH', 'None')
 checkpoint = os.environ.get('CHECKPOINT_PATH', 'None')
 
-try:
-    with open(config_path) as f:
-        config = yaml.safe_load(f)
-    frame_shape = config['dataset_params']['frame_shape']
-    generator_params = config['model_params']['generator_params']
-    use_lr_video = generator_params.get('use_lr_video', False)
-    lr_size = generator_params.get('lr_size', 64)
-    generator_type = generator_params.get('generator_type', 'occlusion_aware')
-except Exception as e:
-    frame_shape = [1024, 1024, 3]
-    use_lr_video = False
-    generator_type = 'vpx'
+main_configs = get_main_config_params(config_path)
```

Comment:
same comment as above

Reply:
I PRed the branch in nets_implementation

Reply:
Can you first approve https://github.mit.edu/NeTS/nets_implementation/pull/41 so that we can merge those into master and then replay the new commits from here ontop of the new master so that we don't have redundant commits in this.

Reply:
done